back-end/controllers/response.controller.js

```diff
@@ -0,0 +1,20 @@
+const responseService = require('../services/response.service');
+
+module.exports = {
+    addResponse,
+    getResponsesforQuestion,
+};
+
+// req, res, next follows express.js syntax - lookup for info
+function addResponse(req, res, next) {
+    responseService.addResponse(req.body)
+        .then(() => res.status(201).send({}))
```

Comment:
Something useful for the front-end would be adding the correct response here, if there is one. This would make the ```responseService.addResponse()``` service return either a blank string if no correct answer or a string of the correct answer if one exists, then sending it up ```.then((correctAnswer) => res.status(201).send(correctAnswer))```